modules/ROOT/pages/install-self-managed.adoc

```diff
@@ -2,6 +2,9 @@
 
 You can use the Runtime Fabric command line tool, `rtfctl`, to install Anypoint Runtime Fabric on an Amazon Elastic Kubernetes Service (Amazon EKS), Amazon Elastic Kubernetes Service Anywhere (Amazon EKS-A), Azure Kubernetes Service (AKS) or Google Kubernetes Engine (GKE) installation that you manage. 
 
+[NOTE]
+Runtime Fabric version equal or greater than 2.2.5 with multiple instances support requires `rtfctl` version equal or greater than 0.4.x.
```

Comment:
Can we use  `>=` instead of words?

Reply:
and not 0.4.x should be `>0.4`  as we have `0.4.1` as first GA patch version for 0.4.x

Reply:
Runtime Fabric versions >= 2.2.5 with multiple instances support require `rtfctl` version > 0.4.